network-api/networkapi/wagtailpages/templates/wagtailpages/homepage.html

```diff
@@ -24,27 +24,42 @@
   {% include "./fragments/cause_statement.html" %}
   {% include "./fragments/focus_areas.html" %}
 </div>
-<div class="bg-gray py-5">
-  <div class="container">
-    {% include "./fragments/news_you_can_use.html" %}
-  </div>
-</div>
-<div class="container">
 
-  {% include "./fragments/spotlight_posts.html" %}
+<div class="container">
 
-</div>
-<div class="bg-gray py-5">
-  <div class="container">
-    {% include "./fragments/take_action.html" %}
```

Comment:
A little confused here - can we keep "take action" template code in `take_action.html` in this PR?

Reply:
Yes, this should not have been removed. Not sure what I was thinking here.